### PR TITLE
connected the moderator homepage to the backend

### PR DESCRIPTION
**Pull request by simarahmtkhy**

### CODE REVIEW

File: app/mobile/pubspec.lock [hunk 1]
```diff
@@ -228,18 +236,18 @@ packages:
     dependency: transitive
     description:
       name: plugin_platform_interface
-      sha256: da3fdfeccc4d4ff2da8f8c556704c08f912542c5fb3cf2233ed75372384a034d
+      sha256: f4f88d4a900933e7267e2b353594774fc0d07fb072b47eedcd5b54e1ea3269f8
```
Comment (ThoAppelsin) [inline app/mobile/pubspec.lock:239]: Is it OK that everybody can see this?

Comment (farukyld) [reply app/mobile/pubspec.lock:239]: yes sir!
[see](https://chat.openai.com/share/04367e1c-acfd-448c-ba19-4ce6d8d47247)

Comment (farukyld) [reply app/mobile/pubspec.lock:239]: yes sir! [see](https://chat.openai.com/share/04367e1c-acfd-448c-ba19-4ce6d8d47247)

Comment (farukyld) [reply app/mobile/pubspec.lock:239]: > Is this safe for everyone to see?

more specifically this [comment](https://github.com/dart-lang/pub/issues/2462#:~:text=In%20either%2C%20case%3A%20,to%20verify%20this%20behavior%20independently)

**Pull request merged**